tests/forte/processors/data_augment/algorithms/eda_processors_test.py

```diff
@@ -0,0 +1,166 @@
+            tgt_pack.set_text(src_pack.text)
+            for anno in src_pack.get(Annotation):
+                new_anno = type(anno)(
+                    tgt_pack, anno.begin, anno.end
+                )
+                tgt_pack.add_entry(new_anno)
+
+            swap_processor._process(m_pack)
+
+            new_src_pack = m_pack.get_pack('augmented_input_src')
+
+            self.assertEqual(new_src_pack.text, expected_text)
+            for j, token in enumerate(new_src_pack.get(Token)):
+                self.assertEqual(token.text, expected_tokens[j])
+
+    @data((["Mary and Samantha arrived at the bus station early but waited until noon for the bus."],))
+    @unpack
+    def test_random_insert(self, texts):
+        for idx, text in enumerate(texts):
+            file_path = os.path.join(self.test_dir, f"{idx + 1}.txt")
+            with open(file_path, 'w') as f:
+                f.write(text)
+
+        expected_text = \
+            "await Mary and Samantha arrived at the bus station early but waited until noon for the bus.\n"
+        expected_tokens = ['await ', 'Mary', 'and', 'Samantha', 'arrived', 'at', 'the', 'bus', 'station', 'early',
+                           'but', 'waited', 'until', 'noon', 'for', 'the', 'bus', '.']
+
+        insert_processor = RandomInsertionDataAugmentProcessor()
+        insert_processor.initialize(resources=None, configs=insert_processor.default_configs())
+
+        for idx, m_pack in enumerate(self.nlp.process_dataset(self.test_dir)):
+            src_pack = m_pack.get_pack('input_src')
+            tgt_pack = m_pack.get_pack('output_tgt')
+
+            # Copy the source pack to target pack.
+            tgt_pack.set_text(src_pack.text)
+            for anno in src_pack.get(Annotation):
+                new_anno = type(anno)(
+                    tgt_pack, anno.begin, anno.end
+                )
+                tgt_pack.add_entry(new_anno)
+
+            insert_processor._process(m_pack)
+
+            new_src_pack = m_pack.get_pack('augmented_input_src')
+
+            self.assertEqual(new_src_pack.text, expected_text)
+            for j, token in enumerate(new_src_pack.get(Token)):
+                self.assertEqual(token.text, expected_tokens[j])
+
+    @data((["Mary and Samantha arrived at the bus station early but waited until noon for the bus."],))
+    @unpack
+    def test_random_delete(self, texts):
+        for idx, text in enumerate(texts):
+            file_path = os.path.join(self.test_dir, f"{idx + 1}.txt")
+            with open(file_path, 'w') as f:
+                f.write(text)
+
+        expected_text = \
+            "Mary and   at  bus   but waited until  for the .\n"
+        expected_tokens = ['Mary', 'and', 'at', 'bus', 'but', 'waited', 'until', 'for', 'the', '.']
+
+        delete_processor = RandomDeletionDataAugmentProcessor()
+        processor_config = delete_processor.default_configs()
+        processor_config.update({"alpha": 0.5})
```

Comment:
This is not how you use the configs. You just need to pass in those new parameters. The system will merge them with the default config.